lib/Common/DataStructures/Chakra.Common.DataStructures.vcxproj

```diff
@@ -29,7 +29,7 @@
     </ClCompile>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <ClCompile Include="$(MSBuildThisFileDirectory)BigInt.cpp" />
+    <ClCompile Include="BigUInt.cpp" />
```

Comment:
Is there any particular reason you removed `$(MSBuildThisFileDirectory)` here? Unless it was causing a problem, I'd prefer to keep it so that this line is consistent with the ones around it.

Reply:
VS removes it automatically (or rather, doesn't add it). This needs to be fixed up manually in the vcxproj file for any files added so that ChakraCore project files work in our other build |systems.